packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import classnames from 'classnames';
+import ExpandCollapsePanel, {
+  ExpandCollapsePanelProps,
+  PanelTrigger
+} from '../ExpandCollapsePanel';
+import randomId from '../../utils/rndid';
+
+export interface AccordionTriggerProps
+  extends React.HTMLAttributes<HTMLButtonElement> {
+  className?: string;
+  children: React.ReactElement;
+}
+
+const AccordionTrigger = ({ children }: AccordionTriggerProps) => {
+  return <>{children}</>;
+};
+
+interface AccordionContentProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode | React.ReactNode[];
+  className: string;
+}
+
+const AccordionContent = ({
+  children,
+  className,
+  ...otherProps
+}: AccordionContentProps) => {
+  return (
+    <div className={classnames('Accordion__panel', className)} {...otherProps}>
```

Comment:
Nitpick: This one is a bit weird because it creates a double nested div:

```html
<div class="ExpandCollapse__panel" id="...">
  <div class="Accordion__panel">Here is some content</div>
</div>
```

If it's something we can easily solve it would be nice to not have div soup here, but I'm not going to die on that hill.

Reply:
I tried doing this briefly, mainly through use of a fragment on the `<AccordionContent>`, but it ended up causing issues with the animation (it would no longer animate expand, only collapse). What would you think about leaving for now and adding it into the advance accordion issue as an item to address?

Reply:
Ended up being able to work it out using the fragment and a little more specificity with the CSS for the element!